dojo/settings/settings.dist.py

```diff
@@ -40,6 +40,7 @@
     DD_SECURE_CONTENT_TYPE_NOSNIFF=(bool, True),
     DD_CSRF_COOKIE_SAMESITE=(str, 'Lax'),
     DD_SESSION_COOKIE_SAMESITE=(str, 'Lax'),
+    DD_APPEND_SLASH=(bool, False),
```

Comment:
We should default to True here to match what it has always been for folks as well as the default set by Django